config/stampede_intel.py

```diff
@@ -37,9 +52,22 @@
 #
 #--------------------------------------------------
 
-charm_path   = home + '/Charm/682/intel/omni/charm'
-papi_inc = home + '/include'
-papi_lib = home + '/lib'
+if os.path.isdir(home + '/Charm/682/intel/omni/charm'):
```

Comment:
`charm_path_search` and `grackle_path_search` could take arguments for additional paths to search first. That would eliminate the extra if statements.